src/components/byline.tsx

```diff
@@ -103,11 +103,18 @@ const getAnchorStyles = (format: Format): SerializedStyles => {
     }
 }
 
+const getProfileLink = (node: Node): string => {
+    const href = getHref(node).withDefault('');
+    return href.startsWith('profile/')
+        ? `https://www.theguardian.com/${href}`
```

Comment:
Without this does it point to: `https://mobile.guardianapis.com/profile/journalist`? The apps presumably don't intercept MAPI URLs?

Reply:
Yeah the apps seem to handle profile links on dotcom but not from mapi. Think it makes sense to make the update here rather than in two places!

Really I think capi should be returning the absolute link in this case.